path_config.go

```diff
@@ -30,7 +30,6 @@ func pathConfig(b *backend) *framework.Path {
 			},
 		},
 		Callbacks: map[logical.Operation]framework.OperationFunc{
-			logical.CreateOperation: b.pathConfigWrite,
```

Comment:
Why can this be removed?

Reply:
It didn't seem to be necessary. But I've just read through some of the hashicorp/vault repo and the general pattern seems to be to include both. I'll adjust this.